src/components/SelectionBox.tsx

```diff
@@ -46,14 +45,26 @@ export interface SelectionBoxProps {
 	style?: StyleProp<object>; // TODO: update generic from "object"
 }
 
+const enabledFiltersMap = [
+	{ label: "Restaurants", value: "restaurant" },
+	{ label: "Bars", value: "bar" },
+	{ label: "Drugstores", value: "drugstore" },
+	{ label: "Supermarkets", value: "supermarket" },
+	{ label: "Banks", value: "bank" },
+	{ label: "Museums", value: "museum" },
+	{ label: "ATMs", value: "atm" },
+	{ label: "Bakeries", value: "bakery" },
+	{ label: "Laundromats", value: "laundry" },
+] as const;
```

Comment:
what does casting this as a `const` do?

Reply:
it's actually not `const` in the same sense as `const somename = []`, in TS the `as const` assertion basically makes an object or array un-changeable. So normally `const somename = []` you would still be able to do `somename.push("test")` but with `const somename = [] as const` you would get a type error if you try to do `somename.push`

Reply:
ah I see makes sense!